resources/build/tests/trap-test-builds.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
Maybe I should rename this file to `trap-test-utils.sh`?  It's meant as a utility script to provide simple 'reproductions' of scenarios to test in the actual test script `build-utils-traps.test.sh`.